dpdd.tex

```diff
@@ -68,7 +68,7 @@
 \setDocAbstract{%
 This document describes the data products and processing services to be delivered by the Large Synoptic Survey Telescope (LSST).
 
-The LSST will deliver three levels of data products and services. \textbf{Prompt} (Level 1) data products will include images, difference images, catalogs of sources and objects detected in difference images, and catalogs of Solar System objects. Their primary purpose is to enable rapid follow-up of time-domain events. \textbf{Data Release} (Level 2) data products will include well calibrated single-epoch images, deep coadds, and catalogs of objects, sources, and forced sources, enabling static sky and precision time-domain science. \textbf{User Generated} (Level 3) data product services will enable science cases that greatly benefit from co-location of user processing and/or data within the LSST Archive Center. LSST will also devote 10\% of observing time to programs with special cadence. Their data products will be created using the same software and hardware as Prompt (Level 1) and Data Release (Level 2) products. All data products will be made available using user-friendly databases and web services.
+LSST will deliver three levels of data products\footnote{Data products were referred to as Level 1, Level 2 and Level 3, this nomenclature was updated in 2018 to Prompt Products, Data Release Products and User Generated Products respectively. In the abstract of this document both nomenclatures are used but throughout the remainder of this document only the new terminology is used. In project and requirements documentation the old terminology will likely persist.} and services. \textbf{Prompt} (Level 1) data products will include images, difference images, catalogs of sources and objects detected in difference images, and catalogs of Solar System objects. Their primary purpose is to enable rapid follow-up of time-domain events. \textbf{Data Release} (Level 2) data products will include well calibrated single-epoch images, deep coadds, and catalogs of objects, sources, and forced sources, enabling static sky and precision time-domain science. \textbf{User Generated} (Level 3) data product services will enable science cases that greatly benefit from co-location of user processing and/or data within the LSST Archive Center. LSST will also devote 10\% of observing time to programs with special cadence. Their data products will be created using the same software and hardware as Prompt (Level 1) and Data Release (Level 2) products. All data products will be made available using user-friendly databases and web services.
```

Comment:
Grammar: "Level 2 and Level3, this" -> "Level 2, and Level 3. This" or possibly "Level 2, and Level 3; this".